420 classic dyalog mode (#958)
Dyalog/documentation: windows-ui-guide/docs/session-menubar.md

## The Layout Menu

> The *Layout* menu (`⎕SE.mb.layout`) selects between Standard and Classic mode, and, in Standard mode, the position of the Debugger. See [The Editor](editor.md) for the window management that each mode provides.

Reviewer: ```suggestion
The *Layout* menu (`⎕SE.mb.layout`) selects between Standard and Classic Dyalog mode, and, in Standard mode, the position of the Debugger. See [The Editor](editor.md) for the window management that each mode provides.
```

Table: Layout Menu Operations {: #debugger-layout-menu-operations }

|Item                   |Action             |Description                                        |
|-----------------------|-------------------|---------------------------------------------------|

> |Classic                |&nbsp;             |Selects Classic mode, in which Edit and Trace windows are free-floating|

Reviewer: ```suggestion
|Classic                |&nbsp;             |Selects Classic Dyalog mode (Edit and Trace windows are free-floating)|
```
